src/python/pants/backend/jvm/tasks/bootstrap_jvm_tools.py

```diff
@@ -70,7 +70,9 @@ def __eq__(self, other):
     return type(self) == type(other) and self._tuple() == other._tuple()
 
 
-class BootstrapJvmTools(IvyTaskMixin, CoursierMixin, JarTask):
+# NB: IvyTaskMixin conflicts with the resolve() method of CoursierMixin. IvyTaskMixin.resolve()
+#  will be used because it appears first in the MRO.
```

Comment:
Is it typical to add a leading space here? What is the style being applied here?

Reply:
Also, thank you so much for adding this comment!!!

Reply:
Eh, less typical than not but I've seen others do it. I'll go back to normal.